diff -N -U 3 -r x264-20100106-orig/configure x264-20100106/configure
--- x264-20100106-orig/configure	2010-01-06 13:06:47.000000000 +0200
+++ x264-20100106/configure	2010-01-06 13:16:22.000000000 +0200
@@ -460,7 +460,7 @@
     # and it doesn't save any speed anyway on 4.4, so disable it
     CFLAGS="-O4 -fno-fast-math $CFLAGS"
 else
-    CFLAGS="-O4 -ffast-math $CFLAGS"
+    CFLAGS="$CFLAGS -O4 -ffast-math"
 fi
 
 if cc_check "stdio.h" "" "fseeko(stdin,0,0);" ; then
